cadc-vosi/src/main/java/ca/nrc/cadc/vosi/AvailabilityServlet.java

```diff
@@ -55,22 +124,28 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
             log.info(logInfo.start());
 
             Class wsClass = Class.forName(pluginClassName);
-            WebService ws = (WebService) wsClass.newInstance();
+            AvailabilityPlugin ws = (AvailabilityPlugin) wsClass.newInstance();
```

Comment:
the variable 'ws' doesn't make much sense anymore... ap or availabilityPlugin instead?

Reply:
Changed.